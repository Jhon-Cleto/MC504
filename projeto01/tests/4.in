4 12
10 5
5342