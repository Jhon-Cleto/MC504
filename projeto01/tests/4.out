Erro: Matrizes com dimensÃµes invalidas!
